LadybugTools_Engine/Python/src/ladybugtools_toolkit/honeybee_extension/sri.py

```diff
@@ -1,9 +1,18 @@
 """Methods for calculating the Solar Reflective Index (SRI) of materials and constructions."""
 
 import numpy as np
+import pandas as pd
+from honeybee.typing import clean_string
 from honeybee_energy.construction.opaque import OpaqueConstruction
 from honeybee_energy.material.opaque import EnergyMaterial
-from python_toolkit.bhom.analytics import bhom_analytics
+
+from honeybee_radiance.modifier.material import Plastic
+from matplotlib.colors import hex2color
+from sklearn.linear_model import LinearRegression
+
+from .. import SRI_DATA
+from ..bhom.analytics import bhom_analytics
+from ..bhom.logging import CONSOLE_LOGGER
```

Comment:
```suggestion
from python_toolkit.bhom.analytics import bhom_analytics
from python_toolkit.bhom.logging import CONSOLE_LOGGER
```
The PR merged yesterday moved bhom_analytics and the console logger to Python_Toolkit,
I'll leave the rest to @CKBoulter